### PR TITLE
add player checks

### PR DESCRIPTION
**Pull request by Ramiels**

### CODE REVIEW

File: files/content/coveryourselfinoil/effect.lua [hunk 1]
```diff
@@ -5,7 +5,9 @@ if root and root ~= entity_id and GameGetGameEffectCount(root, "WET") > 0 then
 	local controls_comp = EntityGetFirstComponentIncludingDisabled(root,"ControlsComponent")
 	if not controls_comp or ComponentGetValue2(controls_comp, "mButtonDownDown")then return end;
 	if(not character_data_comp)then return end
-	GameAddFlagRun("oiled_up")
+	if EntityHasTag(root, "player_unit") then
```
Comment (dextercd) [inline files/content/coveryourselfinoil/effect.lua:8]: Should it check for polymorphed_player tag too?

**Comment by dextercd** [comment]
"Avoided Heart Attack!" can be obtained while you're waiting in the game over screen. Not sure if you want to keep that

File: files/content/coveryourselfinoil/effect.lua (hunk 1 above)
Comment (Ramiels) [reply files/content/coveryourselfinoil/effect.lua:8]: Good point, will add

**Comment by Ramiels** [comment]
> "Avoided Heart Attack!" can be obtained while you're waiting in the game over screen. Not sure if you want to keep that

Ah I hadn't considered that the code runs on the menu, I suppose I could check for if the player is alive?

**Comment by dextercd** [comment]
Awesome

**Pull request merged**